pytom/agnostic/score.py

```diff
@@ -12,9 +12,9 @@ def Vol_G_Val(volume, value):
     """
     Vol_GE_Val: returns True when peak in volume greater than value
     @param volume: A volume
-    @type volume: L{pytom_volume.vol}
+    @type volume: L{pytom.lib.pytom_volume.vol}
```

Comment:
```suggestion
    @type volume: L{xp.ndarray}
```

This function is only used in agnostic and should only work for numpy and cupy arrays. See also changes below.